example/gluon/word_language_model/train.py

```diff
@@ -125,19 +119,19 @@ def train():
             hidden = detach(hidden)
             with autograd.record():
                 output, hidden = model(data, hidden)
-                L = loss(output, target)
+                L = mx.nd.mean(loss(output, target))
```

Comment:
Should be fine. Just want to make the parameter consistent with other package, e.g. pytorch. I can update it (need to rescale the clipping and learning rate).